clang/include/clang/Driver/Driver.h

```diff
@@ -602,6 +602,16 @@ class Driver {
   // FIXME: This should be in CompilationInfo.
   std::string GetProgramPath(StringRef Name, const ToolChain &TC) const;
 
+  /// GetModuleManifestPath - Lookup the name of the Standard library manifest.
+  ///
+  /// \param C - The compilation.
+  /// \param TC - The tool chain for additional information on
+  /// directories to search.
+  //
+  // FIXME: This should be in CompilationInfo.
+  std::string GetModuleManifestPath(const Compilation &C,
```

Comment:
Could this be something fallible instead? Printing "" for GNU is suboptimal.

Reply:
That is an option, I could do `<NOT FOUND>` or something like that. I'm not sure what the tool vendors prefer. A magic string or an empty string.